src/newItinerary/HiddenStops.tsx

```diff
@@ -0,0 +1,46 @@
+import React, { useState } from 'react'
+
+import { A11yProps, pickA11yProps } from '../_utils/interfaces'
+import { Bullet, BulletTypes } from '../bullet'
+import { HiddenStopProps } from './HiddenStop'
+import { StyledHiddenStops, StyledStopsCount, StyledWrapper } from './HiddenStops.style'
+import { Line, LineProps } from './internals/Line'
+
+export type HiddenStopsProps = A11yProps &
+  LineProps &
+  Readonly<{
+    children: React.ReactElement<HiddenStopProps>[]
+    label: string
+  }>
+
+export const HiddenStops = ({
+  prevLine,
+  nextLine,
+  children,
+  label,
+  ...props
+}: HiddenStopsProps) => {
+  const [hidden, setHidden] = useState(true)
+
+  return (
+    <StyledHiddenStops
```

Comment:
After testing the collapse part with VoiceOver, I remark it was read 2 times and the UI doesn't allow us to follow those W3C example https://www.w3.org/WAI/GL/wiki/Using_the_WAI-ARIA_aria-expanded_state_to_mark_expandable_and_collapsible_regions as the toggle button disappear after click.
So I suggest to keep the hide/show logic for visual user only and let the sublist alway readable by screen reader, without any button purpose.
So I suggest to do:
- On `StyledHiddenStops ` remove `role="button"`
- On `StyledStopsCount as="button"` add `aria-hidden="true"`
- On the sublist `<ul>` remove `aria-hidden={hidden}`